resources/lib/bom/bom_radar.py

```
@@ -288,12 +290,18 @@ def build_images(radar_code, path, loop_path):
                     Logger.debug("Output to file: " + output_file)
 
                     try:
-                        radar_image = urllib.request.urlopen(image_to_retrieve)
-                        with open(loop_path + "/" + output_file, "wb") as fh:
-                            fh.write(radar_image.read())
+                        with urllib.request.urlopen(image_to_retrieve, timeout=15) as radar_image:
```

Ruff (0.13.1)

Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)